shared/src/commonMain/kotlin/org/hyperskill/app/likes/domain/model/LikeValue.kt

```diff
@@ -1,6 +1,5 @@
 package org.hyperskill.app.likes.domain.model
 
 enum class LikeValue(val value: Int) {
-    VALUE_0(0),
-    VALUE_1(1)
+    ABUSE_COMMENT(1)
```

Comment:
Technically `VALUE_1` can be used with different subjects and depends on the subject that value means different things, so `ABUSE_COMMENT` is only true when a subject is COMMENT.

<img width="607" alt="Screenshot 2023-06-06 at 1 55 33 PM" src="https://github.com/hyperskill/mobile-app/assets/8586063/ed8a0703-1ac8-4f43-8a5a-d3475bef7646">

I propose you revert the change and earlier I asked you to just add some documentation comment about when `VALUE_1` can be used in the project.